src/tainting/Dataflow_tainting.ml

```diff
@@ -435,48 +435,224 @@ let partition_mutating_sources sources_matches =
   |> List.partition (fun (m : R.taint_source tmatch) ->
          m.spec.source_by_side_effect && is_exact m)
 
+let propagate_taint_via_java_setter env e args all_args_taints =
```

Comment:
Has this function also changed? It would be great if this kind of changes weren't mixed with moving code around because it makes it a lot harder to review.